src/main/java/teammates/common/datatransfer/FeedbackQuestionAttributes.java

```diff
@@ -52,9 +55,20 @@ public FeedbackQuestionAttributes(FeedbackQuestion fq) {
         this.showGiverNameTo = new ArrayList<FeedbackParticipantType>(fq.getShowGiverNameTo());
         this.showRecipientNameTo = new ArrayList<FeedbackParticipantType>(fq.getShowRecipientNameTo());
         
+        this.createdAt = fq.getCreatedAt();
+        this.updatedAt = fq.getUpdatedAt();
+        
         removeIrrelevantVisibilityOptions();
     }
 
+    public Date getCreatedAt() {
+        return createdAt;
+    }
+    
+    public Date getUpdatedAt() {
+        return updatedAt;
+    }
```

Comment:
I was following the changes from #4657. One reason the getters are needed might be for JSTL but since it is not needed, I might be able to remove these getters.